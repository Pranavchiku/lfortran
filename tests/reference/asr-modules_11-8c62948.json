{
    "basename": "asr-modules_11-8c62948",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/modules_11.f90",
    "infile_hash": "3adf54a582ee98feab0583ed3cb0093df888f323e87e2cb7defcef52",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-modules_11-8c62948.stdout",
    "stdout_hash": "e330ae227edda2aa66a03fc6738b28939c6097350b7000baca0a55c2",
    "stderr": "asr-modules_11-8c62948.stderr",
    "stderr_hash": "bb71f563df75f300747c95b67e2d57cb079fd95e1951c8a46586739e",
    "returncode": 0
}