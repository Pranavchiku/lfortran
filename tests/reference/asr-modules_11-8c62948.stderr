warning: Implied save feature
 --> tests/../integration_tests/modules_11.f90:3:1
  |
3 | integer :: i = 1
  | ^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/modules_11.f90:4:1
  |
4 | integer :: j = 2
  | ^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
